package.json

```diff
@@ -20,6 +20,7 @@
     "start:platform": "cross-env SINGLE_START=platform npm run dev",
     "start:bpmn": "cross-env SINGLE_START=bpmn npm run dev",
     "start:templates": "cross-env SINGLE_START=templates npm run dev",
+    "start:cloud-templates": "cross-env SINGLE_START=cloud-templates npm run dev",
```

Comment:
Could the start script have more examples in the XML (compare the BPMN for `start:templates`) (e.g., entriesVisible, outdated template, missing template, update template)?

Reply:
Sure, I'll do that.

Reply:
I adjusted it via https://github.com/bpmn-io/bpmn-js-properties-panel/pull/555/commits/e1bdf33d4dff6f48f290a6abf17493f4199afa40

Reply:
Thanks, this makes the fuzzy testing so much easier <3